### ISSUE TITLE
Overall styling

### ISSUE THREAD

**Issue by TonyStark-19** [yash19sinha/coffee-bean#110]
I recently came across your coffee app repository, and I must say, it's a really interesting project with great potential. However, I noticed that the overall styling could benefit from some enhancements to improve the user experience and visual appeal. The current design could be refined in areas such as layout, color scheme, typography, and responsiveness.

Would you still be open to contributors? I would love to help with improving the app's styling and potentially work on other design-related aspects if you're interested.

Looking forward to hearing from you!

**Comment by yash19sinha** [comment]
Ok go ahead, looking forward to some cool changes from your side

**Comment by TonyStark-19** [comment]
I’ve just created a pull request ([#111 ](https://github.com/yash19sinha/coffee-bean/issues/110#PR_LINK)) for the changes I made. While working on the project, I noticed that the readability and maintainability of the code were quite challenging, which made it harder to navigate and make updates. It seems that some contributors may not have followed your rule of maintaining clean, organized code.

I spent some extra time formatting the code to improve its readability and structure, which I believe will make it easier for future contributors to work on. I hope these improvements help, and I’m happy to discuss or adjust anything further if needed.

Thank you for this opportunity
